subekashi/templates/subekashi/base/menu.html

```diff
@@ -1,12 +1,12 @@
 {% load static %}
-<section>
-    <div id="pages">
-        <p>メニュー</p><br />
-        <a class="nokanji" href="{% url 'subekashi:top' %}">トップ</a><br />
-        <a class="nokanji" href="{% url 'subekashi:search' %}">一覧と検索</a><br />
-        <a class="nokanji" href="{% url 'subekashi:new' %}">登録と編集</a><br />
-        <a class="nokanji" href="{% url 'subekashi:make' %}">歌詞の生成</a><br />
-        <a class="nokanji" href="{% url 'subekashi:ai' %}">生成された歌詞</a><br />
-        <a class="nokanji" href="https://www.youtube.com/@subeteanatanoseidesu">全てあなたの所為です。</a><br />
-    </div>
-</section>
\ No newline at end of file
+<nav class="menuarticle" id="menuarticle">
+    <h2 class="nokanji">メニュー</h2>
+    <ul>
+        <li><a class="nokanji" href="{% url 'subekashi:top' %}">トップ</a>
+        <li><a class="nokanji" href="{% url 'subekashi:search' %}">一覧と検索</a>
+        <li><a class="nokanji" href="{% url 'subekashi:new' %}">登録と編集</a>
+        <li><a class="nokanji" href="{% url 'subekashi:make' %}">歌詞の生成</a>
+        <li><a class="nokanji" href="{% url 'subekashi:ai' %}">生成された歌詞</a>
+        <li><a class="nokanji" href="https://www.youtube.com/@subeteanatanoseidesu">全てあなたの所為です。</a>
```

Comment:
liタグが閉じられていません。修正案は以下の通りです。
```suggestion
        <li><a class="nokanji" href="{% url 'subekashi:top' %}">トップ</a></li>
        <li><a class="nokanji" href="{% url 'subekashi:search' %}">一覧と検索</a></li>
        <li><a class="nokanji" href="{% url 'subekashi:new' %}">登録と編集</a></li>
        <li><a class="nokanji" href="{% url 'subekashi:make' %}">歌詞の生成</a></li>
        <li><a class="nokanji" href="{% url 'subekashi:ai' %}">生成された歌詞</a></li>
        <li><a class="nokanji" href="https://www.youtube.com/@subeteanatanoseidesu">全てあなたの所為です。</a></li>
```